### DISCUSSION TITLE
Resource usage vs allocation on the home dashboard (v4.18.2.2)

### DISCUSSION THREAD

**Discussion by iocoiBR** [apache/cloudstack#9516] [Q&A] [answered]
Hello everyone,

Is there any parameter so that compute and storage cards (among all others that involve resource limits) to respect and show the limits defined for a domain instead of bringing the user's own limit data?

**Comment by rajujith** [comment, marked as answer]
@bigsys-it  I think it is not possible currently, but it could be an improvement ticket.
